packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -744,6 +745,13 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// {@macro flutter.material.Material.clipBehavior}
   final Clip? clipBehavior;
 
+  /// {@template flutter.material.appbar.actionsPadding}
+  /// The padding between the [actions] and the end of the AppBar.
+  ///
+  /// By default, the value of [AppBar.actionsPadding] is 0.0.
+  /// {@endtemplate}
+  final double? actionsPadding;
```

Comment:
Should we provide a `EdgeInsetsGeometry?` to provide more flexibility?

Reply:
I think providing a padding to entire AppBar is not necessary.

> I don't think adjusting left, top or bottom padding of the actions is interesting for the developers

But will it be possible that maybe in some use cases, we want more paddings on top and right of each actions? I cannot think of a concrete scenario for the use cases but just feel that changing it to EdgeInsets can provide with more possibilities. WDYT:) ?